cli/listings.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-target=10.3.10.85:8080/da/rest/listings
+target=10.3.10.85:8080/da/rest
```

Comment:
I think it is not important for now and it will be easier for users to use the tool (they do not need to know the url)